bin/pem

```diff
@@ -29,9 +29,8 @@ commands() {
 
 main() {
     local COMMAND="$1"; shift
-    [ -z "$COMMAND" ] && usage && exit 1
     [ "$COMMAND" = "commands" ] && commands && exit 0
-    [ ! -e "$PEM_DIR/pem-$COMMAND" ] && echo "Unknown command: pem $COMMAND" && usage && exit 2
+    [ ! -e "$PEM_DIR/pem-$COMMAND" -o -z "$COMMAND" ] && echo "Unknown command: pem $COMMAND" && usage && exit 2
```

Comment:
@ivan-podorozhnyi 

1. The issue is still reproducible. 
2. When call `pem`, we have to see `usage` output only.